src/components/MyTemplate/MyTemplateContainer.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable require-jsdoc */
+import React from "react";
+import MyTemplateListItem from "./MyTemplateListItem";
+import { Flex, Text, Button } from "@chakra-ui/react";
+import { IMyTemplates } from "@/interfaces/template";
+import { useDispatch } from "react-redux";
+import { switchMyTemplateDisplay } from "@/store/reducer/buttonToggleSlice";
+
+interface myTemplates {
+  myTemplates: IMyTemplates[] | undefined;
+}
+
+export default function MyTemplateContainer({ myTemplates }: myTemplates) {
+  const disPatch = useDispatch();
+  const handleReturnToDesign = () => {
+    disPatch(switchMyTemplateDisplay(false));
+  };
+  const myTemplateLists = (): JSX.Element[] | undefined => {
+    return myTemplates
+      ?.sort((itemA, itemB) => +itemB.createdAt - +itemA.createdAt)
+      .map((item) => <MyTemplateListItem key={item._id} {...item} />);
+  };
+
+  return (
+    <Flex flexDirection="column" width="1440px" gap="28px" position="relative" alignItems="center">
+      <Text fontSize="32px" fontWeight="700" textAlign="center">
+        My Template
+      </Text>
+      {myTemplateLists()}
```

Comment:
It is very weird to use a function as a const value here...I suggested you to write like: 
const myTemplateLists: JSX.Element[] = data?.map()

Reply:
> 



> It is very weird to use a function as a const value here...I suggested you to write like: const myTemplateLists: JSX.Element[] = data?.map()

myTemplateList is an object so it cannot be written dirrectly.
If I change it to a function, then it can be written dirrenctly.